[bug] make unit-test failed

### Ⅰ. Issue Description
make unit-test has the following error:
```
make unit-test
unit-test
volume_build.go:8:8: cannot find package "github.com/alibaba/pouch/volume/store/boltdb" in any of:
	/tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/alibaba/pouch/volume/store/boltdb (vendor tree)
	/usr/local/go/src/github.com/alibaba/pouch/volume/store/boltdb (from $GOROOT)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/volume/store/boltdb (from $GOPATH)
	/tmp/pouchbuild/src/github.com/docker/libnetwork/Godeps/_workspace/src/github.com/alibaba/pouch/volume/store/boltdb
make: *** [unit-test] Error 1
```

This is because unit-test target should depend on modules.
### Ⅱ. Describe what happened


### Ⅲ. Describe what you expected to happen


### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅴ. Anything else we need to know?


### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/Makefile
+++ b/Makefile
@@ -58,7 +58,7 @@ vet: # run go vet
 	@test -z "$$(./hack/build vet)"
 
 .PHONY: unit-test
-unit-test: pre ## run go test
+unit-test: pre modules ## run go test
 	@echo $@
 	@./hack/build unit-test
 

--- a/hack/build
+++ b/hack/build
@@ -52,6 +52,7 @@ function vet()
 function unit-test()
 {
     cd $BUILDPATH/src/github.com/alibaba/pouch
+    go test -i
     for d in `go list ./... | grep -v 'github.com/alibaba/pouch/test' | grep -v 'github.com/alibaba/pouch/extra' | grep -v 'github.com/alibaba/pouch/vendor' `
     do
         go test -coverprofile=profile.out -covermode=atomic $d

